### PR TITLE
Reintroduce accesspecifier

### PR DESCRIPTION
**Pull request by matozoid**
Reintroduce accesspecifier after some mild complaints.

### CODE REVIEW

File: javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocExtractorTest.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ void canParseAllJavadocsInJavaParser() throws FileNotFoundException {
 
     private void processFile(File file) throws FileNotFoundException {
         try {
-            CompilationUnit cu = parse(file, StandardCharsets.UTF_8);
+            CompilationUnit cu = parse(file);
```
Comment (ftomassetti) [inline javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocExtractorTest.java:45]: Why this?

File: javaparser-core/src/main/java/com/github/javaparser/ast/AccessSpecifier.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package com.github.javaparser.ast;
+
+/**
+ * Access specifier. Represents one of the possible levels of
+ * access permitted by the language.
+ *
+ * @author Federico Tomassetti
+ * @since July 2014
+ */
+public enum AccessSpecifier {
```
Comment (ftomassetti) [inline javaparser-core/src/main/java/com/github/javaparser/ast/AccessSpecifier.java:31]: The good old AccessSpecifier. It is almost five years old!

**Review by ftomassetti — APPROVED** [review]
Nothing surprsing, +1

File: javaparser-core/src/main/java/com/github/javaparser/ast/AccessSpecifier.java (hunk 2 above)
Comment (matozoid) [reply javaparser-core/src/main/java/com/github/javaparser/ast/AccessSpecifier.java:31]: I will never delete it again!

File: javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocExtractorTest.java (hunk 1 above)
Comment (matozoid) [reply javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocExtractorTest.java:45]: A recent PR moved the character encoding to ParserConfiguration, making parse methods with an explicit encoding redundant.

(And the default is UTF8 anyway)

**Pull request merged**